### DISCUSSION TITLE
v0.10.6: Stable Release

### DISCUSSION THREAD

**Discussion by Amazinite** [endless-sky/endless-sky#9838] [Announcements]
This is a stable release, focusing on bug fixes and other small improvements.

These changes include:
* The Gatling Turret and Javelin Turret now each require one crew member.
* The timer ship in "Sad Archie" has been given the "ramming" personality to improve its consistency.
* Animated ship and planet sprites are now correctly rendered in the hail panel instead of not being rendered at all or being rendered at a high frame rate.
* Gave the stock Dreadnought its fourth Torpedo Launcher back that it had lost in a previous release to make room for a bigger battery; it didn't need the bigger battery.

For a complete list of changes, as well as further detail about the summarized changes, see the [changelog](https://github.com/endless-sky/endless-sky/blob/master/changelog). A special thanks to everyone who contributed to this release: @AlexBassett, @alexrovw, @Amazinite, @bene-dictator, @fingolfin, @Koranir, @opusforlife2, @roadrunner56, @Saugia, @tibetiroka, @TomGoodIdea, @warp-core, and @ziproot.

If you are using the high-dpi plugin, you should download and install the [updated version](https://github.com/endless-sky/endless-sky-high-dpi/releases/tag/v0.10.6) that goes with this release.

![image](https://github.com/endless-sky/endless-sky-high-dpi/assets/17688683/af776d58-7dbd-411e-96e0-645731cff094)

<hr /><em>This discussion was created from the release <a href='https://github.com/endless-sky/endless-sky/releases/tag/v0.10.6'>v0.10.6: Stable Release</a>.</em>

**Comment by mariano-f-r** [comment]
Any word on when the Flatpak will be updated?
